dvc/remote/gdrive.py

```diff
@@ -41,30 +42,18 @@ def __init__(self, path):
         )
 
 
-@decorator
-def _wrap_pydrive_retriable(call):
+def gdrive_retry(func):
     from pydrive2.files import ApiRequestError
 
-    try:
-        result = call()
-    except ApiRequestError as exception:
-        retry_codes = ["403", "500", "502", "503", "504"]
-        if any(
-            "HttpError {}".format(code) in str(exception)
-            for code in retry_codes
-        ):
-            raise GDriveRetriableError("Google API request failed")
-        raise
-    return result
-
+    retry_re = re.compile(r"HttpError (403|500|502|503|504)")
```

Comment:
great!, but we'll need to rebuilt it on the PyDrive2 side to get rid of this fragile stuff